### PR TITLE
maxscale should now work on openshift

### PR DESCRIPTION
**Pull request by PetkoVasilev**
changes to how we start maxscale so it works on Openshift

### CODE REVIEW

File: mariadb-enterprise/config/start-maxscale.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```
Comment (alexander-aldev) [inline mariadb-enterprise/config/start-maxscale.sh]: this code should go into docker-entrypoint.sh for the RHEL image

File: mariadb-enterprise/config/start-maxscale.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+# check if running with a fake user (OpenShift environment)
+whoami
+if [ $? -ne 0 ]; then
+    FAKE_USER=1
+fi
+
+set -e
+
+if [ -n $FAKE_USER ]; then
```
Comment (alexander-aldev) [inline mariadb-enterprise/config/start-maxscale.sh]: would prefer a variant of (an additional check to be made whether the UID > 10000 is already present in /etc/passwd

`if [ $(id -u) -ge 10000 ]; then
    cat /etc/passwd | sed -e "s/maxscale:/builder:/" > /tmp/passwd
    echo "maxscale:x:$(id -u):$(id -g):,,,:/home/maxscale:/bin/bash" >> /tmp/passwd
    cat /tmp/passwd > /etc/passwd
    rm /tmp/passwd
fi

/bin/bash "$@"`

File: mariadb-enterprise/catalog/maxscale.yaml [hunk 3]
```diff
@@ -97,13 +97,9 @@ spec:
...
-        args:
-        - maxscale
-        - -d
-        - -U
-        - maxscale
-        - --configdir=/etc/maxscale-cfg
-        - -lstdout
+        command:
+        - bash
+        - /etc/maxscale-cfg/start-maxscale.sh
```
Comment (alexander-aldev) [inline mariadb-enterprise/catalog/maxscale.yaml]: `/usr/bin/docker-entrypoint.sh maxscale -d -U maxscale --configdir=/etc/maxscale-cfg -l stdout`

File: mariadb-enterprise/catalog/mariadb-config.yaml [hunk 4]
```diff
@@ -66,6 +66,10 @@ data:
 {{- else }}
 {{ tpl (.Files.Get "config/start-mariadb-instance.sh") . | indent 4 }}
 {{- end }}
+
```
Comment (alexander-aldev) [inline mariadb-enterprise/catalog/mariadb-config.yaml]: no additional script here, assumed part of docker-entrypoint.sh (should be adjusted in maxscale RHEL image, too)

**Comment by PetkoVasilev** [comment]
after moving the script to the maxscale imaga, there are no changes needed here so I'm closing this PR

**Pull request closed without merging**